haskell/cabal
# Add TARGET option to `cabal check`

Closes #6282 

e.g. `cabal check ~/my-projects/reactive-strawberry/`

---

Include the following checklist in your PR:


---

**QA Notes**

1. `cabal init foo`
2. `cabal check foo`
  - with old cabal, you will get a `'check' doesn't take any extra arguments` error
  - with new cabal you will get a list of warnings
